docker/ray-ml/install-ml-docker-requirements.sh

```diff
@@ -17,10 +19,10 @@ pip --no-cache-dir install -U pip pip-tools
 # Install requirements
 pip --no-cache-dir install -U -r requirements.txt
 
-
 # Install other requirements. Keep pinned requirements bounds as constraints
 pip --no-cache-dir install -U \
            -c requirements.txt \
```

Comment:
Should this file be converted to `-r` now?

Reply:
No, these are still constraints (the `-r` is in line 20). We should keep them here as a sanity check for PRs that may update requirements.txt.